packages/react/src/themes/base/components/Text/textStyles.ts

```diff
@@ -22,7 +21,11 @@ export default {
     },
     variables: v,
   }: ComponentStyleFunctionParam<TextProps, TextVariables>): ICSSInJSStyle => {
+    const colors = v.colorScheme[color] || v.colorScheme['default']
```

Comment:
```suggestion
    const colors = v.colorScheme[getColorSchemeKey(p.color)]
```
?

Reply:
This is in base theme, the function `getColorSchemeKey` is based on colors in teams theme :\ I may introduce a similar method for the base theme as well.

Reply:
Will leave this for now as it is, so at least we can spot the difference between base/teams themes